public/includes/find-house.php

```diff
@@ -0,0 +1,65 @@
+<?php
+require_once('db.php');
+
+function get_states() {
+    $mysqli = db_connection();
+    $sqlQuery = "SELECT * FROM `states` WHERE 1";
+    $result = mysqli_query($mysqli,$sqlQuery);
+    echo "<option selected disabled>State</option>";
+    while ($row = mysqli_fetch_array($result, MYSQLI_ASSOC)) {
```

Comment:
Personally, It would be preferable to return a JSON format as a response from this file (as done with normal API in Nodejs) and build the form on the frontend with Javascript instead of building the form in two places. Also, JSON is lighter and easier to send across a network (that is what it is the preferable standard)